server/src/main/java/io/littlehorse/common/model/getable/core/init/ServerVersionModel.java

```diff
@@ -0,0 +1,44 @@
+package io.littlehorse.common.model.getable.core.init;
+
+import com.google.protobuf.Message;
+import io.littlehorse.common.LHSerializable;
+import io.littlehorse.sdk.common.exception.LHSerdeError;
+import io.littlehorse.sdk.common.proto.ServerVersion;
+import io.littlehorse.server.streams.topology.core.ExecutionContext;
+
+public class ServerVersionModel extends LHSerializable<ServerVersion> {
+
+    private int major_version;
+    private int minor_version;
+    private int patch_version;
+
+    public ServerVersionModel() {}
+    ;
+
+    public ServerVersionModel(int major_version, int minor_version, int patch_version) {
```

Comment:
This is python variable naming. Can you please use `camelCase` since this is Java?